### PR TITLE
Restore running edge in local

### PR DESCRIPTION
**Pull request by vcastellm**
# Description

This PR restore the functionality of `cluster` script for running in local for development and testing

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the relevant labels
- [x] I have updated the official documentation
- [x] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [x] I have tested this code manually

### Manual tests

Running the script

### CODE REVIEW

**Review by goran-ethernal — APPROVED** [review]

File: scripts/cluster [hunk 1]
```diff
@@ -24,7 +28,63 @@ function createGenesis() {
...
-    --burn-contract 0:0x0000000000000000000000000000000000000000
+    --burn-contract 0:0x0000000000000000000000000000000000000000 \
+    --reward-wallet 0xDEADBEEF:1000000
+}
+
+function initRootchain() {
+  echo "Initializing rootchain"
+
+  ./polygon-edge rootchain server > /dev/null &
+  sleep 5
```
Comment (Stefan-Ethernal) [inline scripts/cluster]: Could we ping `http://localhost:8545` (JSON RPC address), instead of waiting predefined amount of time?

**Review by Stefan-Ethernal — APPROVED** [review]
Aside from minor suggestion, LGTM

File: scripts/cluster (hunk 1 above)
Comment (vcastellm) [reply scripts/cluster]: Done in https://github.com/0xPolygon/polygon-edge/pull/1502/commits/9ef5987e2b371a44a07abc5dee60695a7899d7bd

**Pull request merged**